src/components/Checkbox.jsx

```diff
@@ -0,0 +1,79 @@
+import { useState } from "react";
+import styled from "@emotion/styled";
+
+const CheckboxContainer = styled.div`
+  display: none;
+
+  @media (min-width: ${(props) => props.theme.breakpoints.tablet_small}) {
+    display: flex;
+    gap: ${(props) => props.theme.layout.space200};
+    color: ${(props) => props.theme.colors.darkBlue};
```

Comment:
Can we define a new set of variables under line 49 in `themes/light.js` for all the color definitions? This will make it easier to support a dark theme in the future if we have time. Using `theme` variables will also improve consistency and simplify site-wide color changes.

See an example here: https://github.com/unocorn-squad/job-posting-app?tab=readme-ov-file#creating-styled-components-using-emotion.

Reply:
Sure. Just implemented this in today's commit.